girder/utility/path.py

```diff
@@ -44,7 +44,7 @@ def encode(token):
 
 
 def decode(token):
-    """Unescape special characters in a token from a path representation.
+    """Un-escape special characters in a token from a path representation.
```

Comment:
Agreed. I hope there's a Flake8 extension for this, which we could eventually add. For now, it's outside the scope of this PR.